step.yml

```diff
@@ -80,17 +80,36 @@ inputs:
       description: |-
         Environment (cloud service) on which you want to execute your tests.
         Each environment has its own limitation of OS/devices.  Please refer to project batch runs page for available choices.
-      value_options: ["Magic Pod", "Remote TestKit"]
+      value_options: ["Magic Pod", "Remote TestKit", "Remote TestKit Onpremise"]
       is_required: true
       is_expand: true
   - external_service_token: 
     opts:
       title: "External service token"
       description: |-
         Access token to use external cloud services (ex. Remote TestKit) for testing.
-        Required when you have selected **other than Magic Pod** for _Environment_.
+        Required when you have selected **Remote Testkit** for _Environment_.
       is_expand: true
```

Comment:
The original description was **bold** in order to emphasize "other than". If you just want to mark "Remote TestKit" as a value option, please use _italic_ instead of **bold** like `model` parameter. (I think bold is OK, but it's better same mark up is used)